osteensco/fastTravelCLI
# Cmd enhancements

- tests
      - main() test loops over every command
      - install test now automagically runs cleanup.sh as part of cleanup function
- commands
    - rm and rn now prompts user to confirm before removing or renaming
    - rn now fails if user tries to rename key that doesn't exist
    - set now fails if key or path is already saved and notifies user why
- cmdArgs API
    - now includes io.Reader for reading user input
